DiamondLightSource/SynchWeb
# LIMS-1119: Show text on small buttons

**JIRA ticket**: [LIMS-1119](https://jira.diamond.ac.uk/browse/LIMS-1119)

**Summary**:

Some users struggle to request their dewar be dispatched, so lets put some labels on the buttons, if the screen is wide enough.

**Changes**:
- Remove the big "Add Container" button, it is not clear which dewar that adds it to
- Display labels for buttons for "Deactivate", "Print Report", "Add Container", "Dispatch" and "Transfer".
- Make the "Add to Project" button on Samples and Proteins pages have a label as well.

**To test**:
- Go to a shipment that is in the dewar store, check the 4 buttons "Print Report", "Add Container", "Dispatch" and "Transfer" appear with labels.
- Go to a shipment that is "processing", check the extra "Deactivate" button appears with a label.
- Go to a shipment that is at "Stores-out", check the "Dispatch" and "Transfer" buttons don't appear.
- Check each button works as expected ("Print Report" sometimes takes a few attempts)
- Check the labels disappear when the window is less wide.
- Go to a shipment on the i15-1 commissioning proposal, check the "Add Container" button now says "Add Puck".
- Go to the /samples and /proteins pages, check each sample/protein has a link to add to a project with a suitable label, check the button works, check the label disappears if the window gets smaller.
